amr-wind/ocean_waves/relaxation_zones/linear_waves_ops.H

```diff
@@ -73,7 +73,9 @@ struct InitDataOp<LinearWaves>
 
                         phi(i, j, k) = eta - zc;
 
-                        if (phi(i, j, k) + 0.5 * dx[2] >= 0) {
+                        const amrex::Real cell_length_2D =
+                            std::sqrt(std::pow(dx[0], 2) + std::pow(dx[2], 2));
```

Comment:
pow 2 should really be written out. At least that's been my experience... (for performance reasons). Though maybe compilers have gotten better. Still don't trust them ;)

Reply:
so this?
```suggestion
                            std::sqrt(dx[0] * dx[0] + dx[2] * dx[2]));
```